runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/src/aws/smithy/kotlin/runtime/http/engine/okhttp/HttpEngineEventListener.kt

```diff
@@ -16,18 +15,18 @@ import java.net.Proxy
 internal class HttpEngineEventListener(
     private val pool: ConnectionPool,
 ) : EventListener() {
-    private val logger = Logger.getLogger<HttpEngineEventListener>()
-
     private inline fun traceCall(call: Call, crossinline msg: () -> Any) {
-        val sdkRequestTag = call.request().tag<SdkRequestTag>()
-        val sdkRequestId = sdkRequestTag?.execContext?.getOrNull(HttpOperationContext.SdkRequestId)
-        logger.trace { "[sdkRequestId=$sdkRequestId] ${msg()}" }
+        call.request().tag<SdkRequestTag>()?.execContext?.let { ctx ->
+            val sdkRequestId = ctx.getOrNull(HttpOperationContext.SdkRequestId)
```

Comment:
Also related to comment on the aws-sdk-kotlin PR...should there be a child `HTTP` span here? 

IF we were to add this span we may want to make use of [eventListenerFactory](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/-builder/event-listener-factory/) to create an event listener per/call rather than a single static one registered for all calls. This would allow us to extract the span creation once rather than creating child spans per log message.

Reply:
Follow-up: Spin taxonomy of trace spans out into new task.